cmd/podman/cliconfig/config.go

```diff
@@ -416,6 +416,7 @@ type RunlabelValues struct {
 	Quiet           bool
 	SignaturePolicy string
 	TlsVerify       bool
+	Replace         bool
```

Comment:
i try to keep this in alpha sorted order

Reply:
done